src/Configuration.cpp

```diff
@@ -18,7 +18,7 @@
  * along with libusbpp.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include <libusbpp/Configuration.hpp>
+#include <Configuration.hpp>
```

Comment:
This one probably needs to be " instead of < as well, not sure why compiler didn't request this one too.